user/languages/rust.md

```diff
@@ -33,8 +33,13 @@ new to Travis CI please read our [Tutorial](/user/tutorial/) and
 ## Choosing a Rust version
 
 By default, we download and install the latest stable Rust release at the start
-of the build, along with appropriate language tools including `cargo`, `rustc`,
-`rustdoc`, `rust-gdb`, `rust-lldb`, and `rustup`.
+of the build (thanks to `rustup`). The [`minimal` profile][profiles] is used
+and include the following language tools `cargo`, `rustc` and `rustup`.
```

Comment:
```suggestion
and includes the following language tools `cargo`, `rustc`, and `rustup`.
```

Reply:
Thanks @Pezi777. What is the used practice on this repo: do I need to keep it as a new commit? Do I need to rebase and squash it?

Reply:
hey @woshilapin just click on "Commit suggestion" > then I will merge it into master

Reply:
> then I will merge it into master

Please note that travis-ci/travis-build#1799 that implements this change was not yet merged.